### PR TITLE
Trip update consistency

### PR DESCRIPTION
**Pull request by pbougue**
First version testing consistency of trip_update (mostly delays) just before persisting and issuing gtfs-rt

JIRA: http://jira.canaltp.fr/browse/NAVP-395

### CODE REVIEW

File: kirin/core/handler.py [hunk 1]
```diff
@@ -45,6 +45,54 @@ def persist(real_time_update):
...
 
 
+def manage_consistency(trip_update):
+    """
+    receive a TripUpdate, then manage and adjust it's consistency
+    returns False if trip update cannot be managed
+    """
+    current_order = 0
+    previous_stu = None
+    for stu in trip_update.stop_time_updates:
```
Comment (antoine-de) [inline kirin/core/handler.py]: you can do  `for stu, current_order in enumerate(trip_update.stop_time_updates):`

File: kirin/core/handler.py [hunk 2]
```diff
@@ -45,6 +45,54 @@ def persist(real_time_update):
...
+        # rejections
+        if stu.order != current_order:
+            return False
```
Comment (antoine-de) [inline kirin/core/handler.py]: I think we have to log the bad cases to be able to trace them

File: kirin/core/handler.py [hunk 3]
```diff
@@ -45,6 +45,54 @@ def persist(real_time_update):
...
+
+        if not stu.arrival and not stu.departure:
+            return False
+
+        # modifications
+        if not stu.arrival:
+            stu.arrival = stu.departure
+            if not stu.arrival_delay:
+                stu.arrival_delay = stu.departure_delay
```
Comment (antoine-de) [inline kirin/core/handler.py]: same here, I think we need to log when we change the results

**Comment by pbougue** [comment]
Jenkins, test this please

**Pull request merged**